windows/ReactNativeWebView/ReactWebView2.h

```diff
@@ -24,7 +18,8 @@ namespace winrt::ReactNativeWebView::implementation {
     class ReactWebView2 : public ReactWebView2T<ReactWebView2> {
     public:
         ReactWebView2(Microsoft::ReactNative::IReactContext const& reactContext);
-        void SetMessagingEnabled(bool enabled);
+        void MessagingEnabled(bool enabled);
+        bool MessagingEnabled();
```

Comment:
nit, since these are simple setter/getter, you can mark them noexcept/const where possible so the compiler can generate better code:

```suggestion
        void MessagingEnabled(bool enabled) noexcept;
        bool MessagingEnabled() const noexcept;
```